fuzzy/docker/level1/Dockerfile.ubuntu20.08-python

```diff
@@ -0,0 +1,59 @@
+FROM verificarlo/verificarlo:latest
```

Comment:
@yohanchatelain could we specify a tag/version for verificarlo, just to make it clear which version we're building from?

Reply:
The last tag (v.0.4.0) is 52 commits behind. There was several improvements since then. We can wait for the new tag.

Reply:
I fixed the version of verificarlo to v0.4.1 (latest)